client/src/components/common/Button/index.js

```diff
@@ -16,3 +14,13 @@ export default function Button({
     </div>
   );
 }
+
+Button.propTypes = {
+  value: PropTypes.string,
+  className: PropTypes.string,
+};
+
+Button.defaultProps = {
+  value: 'Login',
+  className: 'button-input',
```

Comment:
the input element already has class button-input, this would duplicate it. Just set it to empty string

Reply:
Done ^_^